src/libData/AccountData/AccountStore.h

```diff
@@ -241,6 +245,15 @@ class AccountStore
 
   std::shared_timed_mutex& GetPrimaryMutex() { return m_mutexPrimary; }
 
+  void IncrementPrimaryWriteAccessCount() { ++m_externalWriters; }
+  void DecrementPrimaryWriteAccessCount() { --m_externalWriters; }
+  bool GetPrimaryWriteAccess() {
+    return m_externalWriters.load() < NUM_OF_WRITERS;
```

Comment:
NUM_OF_WRITERS should be 1 ? since we should allow exclusive write.

Reply:
Changed to 1